### PR TITLE
fix: bearer-ignore-flag not loaded for scan

### PR DESCRIPTION
**Pull request by elsapet**
## Description

We weren't including ignore flags during scan, so bearer.ignore was not being loaded.
Also ensure we look to the right place if scan target is a path

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Comment by elsapet** [comment]
For completeness, we'd probably want to add this flag to bearer actions too (as we have done for `--config-file`). What do you think @gotbadger

**Review by gotbadger — APPROVED** [review]

**Comment by gotbadger** [comment]
maybe move `bearer-ignore-file` to general flags since its used by more than just ignore commands?

File: pkg/commands/ignore.go [hunk 1]
```diff
@@ -263,13 +263,14 @@ $ bearer ignore migrate`,
 			if err != nil {
 				return fmt.Errorf("flag error: %s", err)
 			}
+
 			configFilePath := viper.GetString(flag.ConfigFileFlag.ConfigName)
```
Comment (elsapet) [inline pkg/commands/ignore.go:267]: Curious here: is the following equivalent? 

```suggestion
			configFilePath := viper.GetString(options.GeneralOptions.ConfigFile)
```

We use `flag.ConfigFileFlag.ConfigName` in the commands/scan.go but perhaps this is for performance or other reasons. Any light to be shed @didroe or @cfabianski? 🙏

Comment (didroe) [reply pkg/commands/ignore.go:267]: In this case it's equivalent (without the `viper.GetString`). But in the scan command, we support reading the options from the config file. So we have to read it's location before building the options.

Comment (elsapet) [reply pkg/commands/ignore.go:267]: Ahh, thanks for the explanation. So, if we wanted to support reading options from the config file in this case too, we should leave it as the following?

```golang
configFilePath := viper.GetString(flag.ConfigFileFlag.ConfigName)
```

Comment (didroe) [reply pkg/commands/ignore.go:267]: Yeah. Basically we'd need to do [this](https://github.com/Bearer/bearer/blob/911c5c161774f639f1d34e7dc25a7b4596b8df82/pkg/commands/scan.go#L79-L88) before calling `<FLAGS>.ToOptions`. And therefore you'd need to grab the filename manually first.

**Pull request merged**